packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -122,13 +132,90 @@ export async function builder(yargs) {
       'Set up auth for for SuperTokens',
       (yargs) => standardAuthBuilder(yargs),
       async (args) => {
-        const handler = await getAuthHandler(
+        const { handler } = await getAuthSetup(
           '@redwoodjs/auth-supertokens-setup'
         )
         console.log()
         handler(args)
       }
     )
+    .strict(false)
+    .parserConfiguration({ 'unknown-options-as-args': true })
+    .command(
+      '$0 <npm-package>',
+      'Generate an auth configuration for the given provider',
+      () => {},
+      async () => {
+        // Here be dragons...
+        // There's no way for the user to actually end up here, but we need
+        // this command to get the help output we want from yargs
+      }
+    )
+    .command(
+      '$0',
+      false,
+      () => {},
+      async (args) => {
+        // This is a workaround for https://github.com/yargs/yargs/issues/2291
+        if (args._.length <= 2) {
```

Comment:
Would it ever just not be 2? This handles the case where the user does `yarn rw setup auth` right? Or maybe you're handling `yarn dev` here?